src/SignalR/server/Core/ref/Microsoft.AspNetCore.SignalR.Core.netcoreapp3.0.cs

```diff
@@ -127,9 +127,7 @@ public static partial class HubClientsExtensions
     }
     public partial class HubConnectionContext
     {
-        public HubConnectionContext(Microsoft.AspNetCore.Connections.ConnectionContext connectionContext, System.TimeSpan keepAliveInterval, Microsoft.Extensions.Logging.ILoggerFactory loggerFactory) { }
-        public HubConnectionContext(Microsoft.AspNetCore.Connections.ConnectionContext connectionContext, System.TimeSpan keepAliveInterval, Microsoft.Extensions.Logging.ILoggerFactory loggerFactory, System.TimeSpan clientTimeoutInterval) { }
-        public HubConnectionContext(Microsoft.AspNetCore.Connections.ConnectionContext connectionContext, System.TimeSpan keepAliveInterval, Microsoft.Extensions.Logging.ILoggerFactory loggerFactory, System.TimeSpan clientTimeoutInterval, int streamBufferCapacity) { }
+        public HubConnectionContext(Microsoft.AspNetCore.Connections.ConnectionContext connectionContext, Microsoft.AspNetCore.SignalR.HubOptions hubOptions, Microsoft.Extensions.Logging.ILoggerFactory loggerFactory) { }
```

Comment:
Do we have a standard order for parameters? I.e. do we always try to keep logger parameters last and options parameters second to last?

Reply:
No idea

Reply:
Should this context be the hub options? Why isn't it its own context

Reply:
I chose `HubOption` because it contains the settings we want to pass to `HubConnectionContext` and I didn't want to make a new public type. We can change that if you'd like.

Reply:
Yes I think we should have a `HubConnectionContextOptions` as this will start to break down the minute we need to pass something that isn't exactly an option.